WordPress/Classes/ViewRelated/Cells/SwitchTableViewCell.swift

```diff
@@ -0,0 +1,86 @@
+import Foundation
+
+
+/**
+*  @class           SwitchTableViewCell
+*  @brief           The purpose of this class is to simply display a regular TableViewCell, with a Switch
+*                   on the right hand side.
+*/
+
+public class SwitchTableViewCell : WPTableViewCell
+{
+    // MARK: - Public Properties
+    public var onChange : ((newValue: Bool) -> ())?
+    
+    public var name : String {
+        get {
+            return textLabel?.text ?? String()
+        }
+        set {
+            textLabel?.text = newValue
+        }
+    }
+    
+    public var isOn : Bool {
```

Comment:
Since this is a pass-thru to the switch, maybe just call it "on" to match the backing control?